The command has indeed failed with message:
Obligation 2 already solved.
